### PR TITLE
Load more news

### PR DESCRIPTION
**Pull request by tmccoy529**
Added the Load More news button.

### CODE REVIEW

File: src/components/NewsRoomList.jsx [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
+  const [moreNewsRoomItems, setMoreNewsRoomItems] = useState([]);
+  const {
+    hasError,
+    newsRoomItems = [],
+    isLoading,
+    newsRoomLoadMoreEndPoint
+  } = useNews({ endPoint: "/api/news" });
+
+  const handlesLoadMoreNews = () => {
+    setMoreNewsRoomItems([...newsRoomItems, newsRoomItems]);
```
Comment (martypowell) [inline src/components/NewsRoomList.jsx]: could we do  this concatenation inside of the `useNews` hook?

File: src/hooks/useNews.js [hunk 2]
```diff
@@ -4,23 +4,26 @@ import { GetNews } from "../services/ApiService";
 
 const useNews = props => {
   const [newsRoomItems, setNewsItems] = useState([]);
+  const [newsRoomLoadMoreEndPoint, setNewsRoomLoadMoreEndPoint] = useState([]);
```
Comment (martypowell) [inline src/hooks/useNews.js]: not blocking, but i suspect this hook could just be `loadMoreEndpoint`

**Review by martypowell — CHANGES_REQUESTED** [review]
@tmccoy529 I am getting this when running locally. Is everythying pushed up?

image

**Review by martypowell — APPROVED** [review]

File: src/components/NewsRoomList.jsx (hunk 1 above)
Comment (tmccoy529) [reply src/components/NewsRoomList.jsx]: done

File: src/hooks/useNews.js (hunk 2 above)
Comment (tmccoy529) [reply src/hooks/useNews.js]: done

**Pull request merged**